### ISSUE TITLE
Cache public container image in azure container registry

### ISSUE THREAD

**Issue by bengtfredh** [Altinn/altinn-platform#559]
### Description

Some parts of infrastructure in kubernetes use public container images. Public container images shold be cached in azure container registry. This make sure we are not affected of outages in public container registry like docker.io, gcr.io i.e.

**Comment by ivarne** [comment]
Is this also relevant for the browserless image that is used for PDF?

**Comment by bengtfredh** [comment]
> Is this also relevant for the browserless image that is used for PDF?

Yes, I think the goal must be that all container pulls go to azurecr.io.

**Comment by bengtfredh** [comment]
- https://github.com/Altinn/altinn-studio-charts/pull/45
